app/frontend/src/aquifers/components/View.vue

```diff
@@ -51,6 +56,7 @@
         v-on:cancel="navigateToView"
         :fieldErrors="fieldErrors"
         :record="record"
+        :aquifer_files="aquifer_files"
```

Comment:
Use camelCase for any javascript vars or component properties.  It gets caught by eslint and breaks the syntax highlighting

Reply:
This should be resolved now - along with all the other eslint warnings.